src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  --selected-color: #00acdc;
+
   margin: 0;
+  padding: 0;
+  font-weight: 500;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.logoLink {
+  display: flex;
+  align-items: center;
+
+  .logo {
+    margin-left: 50px;
+    width: 40px;
+    height: 40px;
+  }
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  margin-right: 50px;
```

Comment:
1. it's better to add spaces to header, you have same space from both sides of header
2. those styles for nav do nothing, you don't need them at all